### PR TITLE
[FIX] General English language improvements

### PR DESCRIPTION
**Pull request by HammyHavoc**
:- )

### CODE REVIEW

File: packages/rocketchat-i18n/i18n/en.i18n.json [hunk 1]
```diff
@@ -273,8 +273,8 @@
   "App_status_error_disabled": "Disabled: Uncaught Error",
   "App_status_manually_disabled": "Disabled: Manually",
   "App_status_disabled": "Disabled",
-  "App_author_homepage": "author homepage",
-  "App_support_url": "support url",
+  "App_author_homepage": "Author Homepage",
```
Comment (graywolf336) [inline packages/rocketchat-i18n/i18n/en.i18n.json]: Please revert these changes, they are lowercase on purpose.

Comment (HammyHavoc) [reply packages/rocketchat-i18n/i18n/en.i18n.json]: Just line 276?

Comment (kaiiiiiiiii) [reply packages/rocketchat-i18n/i18n/en.i18n.json]: 277 as well please :)
